### DISCUSSION TITLE
adding rehype and remark plugins

### DISCUSSION THREAD

**Discussion by o-az** [vuejs/vitepress#3938] [Q&A]
How can I add rehype and remark plugins?

In other frameworks I can pass rehype and remark plugins directly:
```ts
export default defineConfig({
  markdown: {
    syntaxHighlight: false,
    rehypePlugins: [
      //
      [rehypePrettyCode, { theme: moonlightTheme }],
    ],
    remarkPlugins: [],
  },
});
```

what's the vitepress way to do this (pass rehype/remark plugins)?

**Comment by brc-dd** [comment]
VitePress uses markdown-it. You cannot use rehype plugins.
